modules/ocf/manifests/networking.pp

```diff
@@ -10,7 +10,7 @@
     $gateway6   = '2607:f140:8801::1',
 
     $domain      = 'ocf.berkeley.edu',
-    $nameservers = ['2607:f140:8801::1:22', '169.229.226.22', '169.229.2.171'],
+    $nameservers = ['2607:f140:8801::1:22', '169.229.226.22', '8.8.8.8'],
```

Comment:
Might be good to mention here that this can only store 3 values at maximum, for anyone changing this in the future?

Reply:
I added an assertion to enforce this.

Reply:
Nice, even better.